publish/binaries/BUILD.bazel

```diff
@@ -46,15 +45,11 @@ BINARIES = {
     "ic-consensus-pool-util": "//rs/artifact_pool:ic-consensus-pool-util",
     "ic-crypto-csp": "//rs/crypto:ic-crypto-csp",
     "ic-nns-init": "//rs/nns/init:ic-nns-init",
-    "ic-prep": "//rs/prep:ic-prep",
```

Comment:
I think UTOPIA downloads ic-prep from the CDN so it has to stay. @michael-weigelt can you confirm?

Reply:
Is there a reason to be on CDN ? Can't we just use `bazel run` from head ?

Reply:
Yes, we need it on the CDN. The reason is that the utopia projects are in different repos in a different org, so we cannot build it ourselves.